english/zoo/zoo.md

```diff
@@ -4,4 +4,7 @@ nature. I saw a lot of tigers, elephants, deer and a whole lot of other [animals
 I was pretty thirsty. There is a lemonade stand over there.
 I head over and buy some [lemonade](../drink/lemonade/lemonade.md)
 
-I choose to get better at [English](../get-better-at-english/english.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.